Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -314,11 +319,69 @@ void CookieStore::remove(CookieStoreDeleteOptions&& options, Ref<DeferredPromise
     set(WTFMove(initOptions), WTFMove(promise));
 }
 
+void CookieStore::cookiesAdded(const String& host, const Vector<Cookie>& cookies)
+{
+    ASSERT_UNUSED(host, host == m_host);
+    if (!m_hasChangeEventListener)
```

Comment:
Shouldn't this be an assertion? I don't think we should be getting cookie notifications if we don't have a listener.